docs/solutions/hypervisors/kvm/kvm-nested/main.tf

```diff
@@ -0,0 +1,61 @@
+provider "oci" {
+  tenancy_ocid     = "${var.tenancy_ocid}"
+  user_ocid        = "${var.user_ocid}"
+  fingerprint      = "${var.fingerprint}"
+  private_key_path = "${var.ssh_private_key_path}"
```

Comment:
ditto

Reply:
Correct - see for example https://github.com/oracle/terraform-provider-oci/blob/master/docs/solutions/omc_monitored_server/env-vars